### ISSUE TITLE
find the file in the positiveControlEnricher.py script

### ISSUE THREAD

**Issue by vguest** [PacificBiosciences/SMRT-Analysis#203]
One line in positiveControlEnricher.py is responsible for the reading of forward or reverse modle that is self.fwd_model = np.genfromtxt("/home/UNIXHOME/obanerjee/initial_lr_model_weights_fwd.csv", delimiter=','). Where can I find the initial_lr_model_weights_fwd.csv and initial_lr_model_weights_fwd.csv files? Thank you!

**Comment by mjhsieh** [comment]
Hi, where is this file mentioned in our document, is it part of our software distribution?

**Comment by RobinQi** [comment]
yes, this is part of software distribution. Located in the current/analysis/bin/positiveControlEnricher.py file.
